zerver/openapi/zulip.yaml

```diff
@@ -12172,6 +12188,10 @@ components:
               (`emoji_code` will be its ID).
             * `zulip_extra_emoji`: Special emoji included with Zulip.  Exists to
               namespace the `zulip` emoji.
+    EmojiReactionBase:
+      allOf:
+        - $ref: "#/components/schemas/EmojiBase"
+      properties:
```

Comment:
Shouldn't these properties be indented at same indentation as the `$ref`? Probably something like:
```
allOf:
  - $ref: "#/components/schemas/EmojiBase"
  - properties:
      user_id:
  ```